superset/contrib/docker/Dockerfile

```diff
@@ -14,7 +14,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM python:3.6-jessie
+FROM node:12 AS static
```

Comment:
node version in build stage and in final image aren't the same. Most probably it's not important but anyway better make them match.

Reply:
Good catch! Fixed [here](https://github.com/src-d/sourced-ui/pull/275/commits/aeffcdb829bd4ff3f890497145d639ddb25cad8a).